player.py

```diff
@@ -518,3 +523,41 @@ def define_allowed_choices_anthill_placement(self, anthill):
       if level is None:
         allowed_choices_placement.append(str(idx))
     return allowed_choices_placement
+
+  def score_wine_in_hand(self):
+    """Calculates player's 'wine' points at the end of the game
+
+    'Wine' points are those that are acquired through the wine tokens 
+    held by the player.
+
+    Could be called any time but to reflect the real game there is no running 
+    total of points; just a single calculation and comparison after the final ant 
+    has reached the anthill.
+
```

Comment:
This function could currently benefit with the text description of how the points are calculated, i.e. "Each wine is worth 1 point for each different type of food you have at least one of". However, if you've chosen not to add this info in anticipation of there being different cards later on and the description will be stored elsewhere that's okay.